Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -2,7 +2,9 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using Microsoft.MixedReality.Toolkit.Input;
+using System.Collections;
 using UnityEngine;
+using UnityEngine.Assertions.Must;
```

Comment:
It looks like these methods are obsolete: https://docs.unity3d.com/2018.4/Documentation/ScriptReference/Assertions.Must.MustExtensions.html. Can we use something else instead?

Reply:
Turns out we didn't need it and it was added in somewhat arbitrarily.